### DISCUSSION TITLE
fooocus for commercial ???

### DISCUSSION THREAD

**Discussion by klncgty** [lllyasviel/Fooocus#2688] [Q&A] [answered]
"Can we use Fooocus for our own commercial venture? What are the terms?"

**Comment by mashb1t** [comment, marked as answer]
Why is this a citation?
Please find more information in https://github.com/lllyasviel/Fooocus/discussions/1957 and clarify with your legal team.

**Reply by mashb1t** [reply]
License: https://github.com/lllyasviel/Fooocus?tab=GPL-3.0-1-ov-file#readme
https://choosealicense.com/licenses/gpl-3.0/

From https://fossa.com/blog/open-source-software-licenses-101-gpl-v3/
> GPL v3 License: The Basics
Like the GPL v2, GPL 3 is a strong [copyleft license](https://fossa.com/blog/all-about-copyleft-licenses/), meaning that any copy or modification of the original code must also be released under the GPL v3. In other words, you can take the GPL 3’d code, add to it or make major changes, then distribute your version. However, your version is subject to the same license requirements, meaning that it must be under GPL v3  as well — anyone can see your modified code and install it for their own purposes.
